styles/components/avatar.css

```diff
@@ -0,0 +1,22 @@
+.avatar {
+  border-radius: 50%;
+  width: 70px;
+  height: 70px;
+  margin: 0.5rem;
```

Comment:
I believe giving margin here might break it somewhere or may produce unexpected results.

Reply:
I actually ave it so that some other content following it should not look too crowded or close

Reply:
Yeah that's also the case, later you may have to override margin here and there.

I hope you understood.